### DISCUSSION TITLE
Share form view with no columns should s... [Sep 01]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3738] [Discord Threads]
> 💁 **o1lab**

**Comment by o1lab** [comment]
> 🙋‍♂️ **michacassola**  
#### Hey <@310157751395287040>  thanks! But my form view has columns in the view and the external party did fill out all that were enabled. But after my described actions it was empty although the view was not changed. After restarting both apps the form worked as expected, no changes to the view itself were made. The column I had deleted was not in the form view.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1012293887806685184)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
